.github/workflows/checks.yml

```diff
@@ -32,6 +32,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+    - uses: pre-commit/action@v3.0.0
```

Comment:
I don't have much experience adding CI jobs. However I believe that this [action](https://github.com/pre-commit/action/blob/main/action.yml) should cause CI to fail if `pre-commit` makes any changes.

Reply:
Great finding!